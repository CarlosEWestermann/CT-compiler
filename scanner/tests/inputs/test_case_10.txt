() >= float! errorhandling {
    int a, b;
    a = 10;
    b = 0;
    if (b != 0) {
        int divisionresult;
        divisionresult = a / b;
        print_result(divisionresult);
    } else {
       return a; 
    };
}
