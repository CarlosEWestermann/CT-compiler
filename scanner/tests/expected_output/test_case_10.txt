1 TK_PR_INT [int]
1 TK_IDENTIFICADOR [a]
1 TK_ESPECIAL [=]
1 TK_LIT_INT [5]
1 TK_ESPECIAL [;]
3 TK_PR_FLOAT [float]
3 TK_IDENTIFICADOR [b]
3 TK_ESPECIAL [=]
3 TK_LIT_INT [10]
3 TK_ESPECIAL [;]
